### ISSUE TITLE
Absolutely dog water hardsuit slowdown balance

### ISSUE THREAD

**Issue by Carmell0** [space-wizards/space-station-14#25754]
## Description

I will go over each offender: 

NSTRA void suit, being rare, fills the role of the best soft suit, effectively locking it at 15% slowdown, meaning going over that makes them useless.

Engineering hardsuits have an ungodly slowdown of 30%. Because engineers are fixing spacing 90% of the time, they usually always wear them, leading to painful slowness. On top of that, other than rad and heat, the damage protection is almost never used. And ON TOP OF that, engis usually use magboots anytime they want to make something which have a whopping 20% more slowdown

Speaking of that, magboots fill a weird role. Sure they make you faster than no gravity, but they also serve as a move speed nerf off station, but at the same time hardsuits also seem to try and do this

Salvage suits are just bad. The slowdown does not justify the reduction. I can see why this is, to nerf salv, which in its own bubble is fairly balanced. But again, it sets precedent for other suits. The mining hardsuit is slower, but feels justified as it's way more protective. 

CMO hardsuit has basically no DR, just some caustic rad and a 5% move speed nerf which is fine BUT it has 10% to all on the head, making it a piece of equipment that I personally believe is free protection. 

Sec suits are all balanced well. I personally think that HoS suit should be the best NT suit outside of ERT gear.

Cap suit feels like it should be better. It's just not as fast nor as protective as most suits, despite being given to the captain

Most antag suits are fine. Not much wrong here. Same with most head suits. 

Basic hardsuit feels balanced when compared to a EVA suit, but it lacks majorly when compared to the others. 

The thing that prompted me to write this is the abomination that is the clown hardsuit. It's faster than most suits, has decent protection (something like an average of 15%) and is made non renewable. The mime suit is worse, patenting the clownsuit and the sec hardsuit helmet for even MORE DR. IMO slightly increased speed vs the EVA that crafts it is justified by the fact clown has to sacrifice his recorder to make it, but the DR is not.

Also, not sure if this is still true, but prisoner EVA is better than regular EVA.

In short, hardsuits need a serious re-evaluation of their speeds and DR. Most of these changes boil down to nerf DR and to buff speed. I personally think decide on a speed value on the EVA suit and work from there.

**Comment by K-Dynamic** [comment]
You've left out normal hardsuits in vaults

Engi hardsuits seem weirdly slow at 30% for their DR, they're best described as bulky rad suits in space (a far cry from their depiction in Dead Space)

For comparison, emergency EVA is 30% (used to be 50%), normal EVA 25%

Magboot speeds are also punishing on top of their usual engi hardsuit pairing - maybe there should be a minimum floor to magboot speed penalty

**Comment by Ilya246** [comment]
magboots don't apply their penalty while disabled
and i think yes, engi should maybe get softsuits which offer no protection of any type but let you move fast

**Comment by K-Dynamic** [comment]
> engi should maybe get softsuits which offer no protection of any type but let you move fast

I had deleted a paragraph about this in my comment earlier

Either they get normal EVA mapped in engineering or get yellow EVA softsuits, similar to paramed EVA but for engis

**Comment by Carmell0** [comment]
Maybe some sort of 2 by 2 Emrgency EVA suit?

**Comment by Kadeo64** [comment]
> Maybe some sort of 2 by 2 Emrgency EVA suit?

emergency EVAs are already 4x4 so an engineer can comfortably carry one in a satchel with no slowdown while also having space for four stacks of materials

**Comment by potato1234x** [comment]
please nerf engi suit speed to 20% or under

**Comment by ThatGuyUSA** [comment]
This reads more like a forum post... going to close it in a few days unless anyone has any reason to contest.

I can't speak for all of the balancing team, not that I am a part of it... but the common consensus seems to be that space faring capabilities did not come without a cost. 

Even suits like the Elite Blood-red Hard Suit #29429 and the Commander Hard suit #35805 which had 0% movement penalty have been giving a 10% movement penalty since this issue was made.

I will be closing it after some time due to its inactivity.
